sql/core/src/test/scala/org/apache/spark/sql/execution/WholeStageCodegenSuite.scala

```diff
@@ -51,6 +51,86 @@ class WholeStageCodegenSuite extends QueryTest with SharedSparkSession
     assert(df.collect() === Array(Row(9, 4.5)))
   }
 
+  test("SPARK-31897: GenerateExec should be included in WholeStageCodegen") {
```

Comment:
nit: please the prefix `SPARK-31897` because this is not a bug.

Reply:
done